### DISCUSSION TITLE
[Features] Block & Layout additional features

### DISCUSSION THREAD

**Discussion by mstfkhazaal** [Z3d0X/filament-fabricator#63] [Ideas]
1- Add search with Icon or Image to Block Select List 
![image](https://user-images.githubusercontent.com/47450090/236001689-3550ef74-0ce2-4d67-8b3f-afde8519a05f.png)
2- Can put Block Global or for specific layout
Example (Blog Templates,Comment Section, Author ) For BlogLayout and Question FAQ Blocks for FAQLayout ...

**Comment by Z3d0X** [comment]
1) Fabricator is just using the "Builder" from Filament, please make this suggestion on the filament repo

2) It is possible using `->hidden()` or `->visible()` on the "block", along with https://filamentphp.com/docs/2.x/forms/fields#using-get-to-access-parent-field-values
